scripts/check-import-linter-config.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+# Update config message
+#:
+#: %c[31;1m =========== Error detected in the import linter config ===========%c[0m
+#: Please update and commit the configuration
+#: %c[30m
+#: $ ./scripts/generate-import-linter-config.py ./.importlinter
+#: $ git add .importlinter
+#: %c[0m
+
+
+diff --color -u .importlinter <(./scripts/generate-import-linter-config.py) \
+  || cat $0 | awk '$1 == "#:" { printf(substr($0, 3) "\n", 27, 27) }'
```

Comment:
This is quite a complicated way of printing the error message :). What about this?

```shell
MSG='\e[31;1m =========== Error detected in the import linter config ===========\e[0m
Please update and commit the configuration
\e[30m
$ ./scripts/generate-import-linter-config.py ./.importlinter
$ git add .importlinter
\e[0m
'

if ! diff --color -u .importlinter <(./scripts/generate-import-linter-config.py) ; then
    printf "$MSG"
    exit 1
fi
```

Reply:
Also, I would add this at the beginning so that the script can be called from any directory:

```bash
cd "$(dirname $0)/.."
```

Reply:
Yes, your solution is much easier to read